Tests/fixtures/smartphone-7.yml

```diff
@@ -9135,3 +9135,273 @@
     model: HEYou 70
   os_family: Android
   browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; U; Android 8.1.0; Digit4G Energy Build/O11019; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/61.0.3163.98 Mobile Safari/537.36 OPR/62.5.2254.61243
+  os:
+    name: Android
+    version: 8.1.0
+    platform: ""
+  client:
+    type: browser
+    name: Opera Mobile
+    version: 62.5.2254.61243
+    engine: Blink
+    engine_version: 61.0.3163.98
+  device:
+    type: smartphone
+    brand: Digit4G
+    model: Energy
+  os_family: Android
+  browser_family: Opera
+-
+  user_agent: Mozilla/5.0 (Linux; U; Android 4.4.4; Digit4G-Pro Build/KTU84P) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 UCBrowser/12.12.9.1226 U3/0.8.0 Mobile Safari/534.30
+  os:
+    name: Android
+    version: 4.4.4
+    platform: ""
+  client:
+    type: browser
+    name: UC Browser
+    version: 12.12.9.1226
+    engine: WebKit
+    engine_version: "534.30"
+  device:
+    type: smartphone
+    brand: Digit4G
+    model: Pro
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; U; Android 8.1.0; JP5 Inmate Media Device Build/3.25.1) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.108 UCBrowser/13.0.0.1288 Mobile Safari/537.36
+  os:
+    name: Android
+    version: 8.1.0
+    platform: ""
+  client:
+    type: browser
+    name: UC Browser
+    version: 13.0.0.1288
+    engine: WebKit
+    engine_version: "537.36"
+  device:
+    type: smartphone
```

Comment:
It seems to be a tablet device.
https://idahoreports.blogs.idahoptv.org/2018/08/22/come-for-the-sentence-stay-for-the-prices/